### DISCUSSION TITLE
Things that are not intuitive

### DISCUSSION THREAD

**Discussion by BAndysc** [BAndysc/WoWDatabaseEditor#28] [General]
Do you find anything in the IDE not intuitive? Do you think something needs more explanation, better visibility? I am willing to hear about it!

**Comment by quassia** [comment]
1. While creating a creature script, can I target it after Guid (Primary key in the Creature table) also?
2. Would be nice to have the DB2 Version for BfA and Shadowlands (Currently available for WotLK, Cataclysm, and Legion).
3. SQL Output has blue color text, is it possible to make it a bit lighter as tone? It's basically impossible to read.
Awesome tool! Thank you for your work.

**Reply by BAndysc** [reply]
Thanks for the feedback!
1. It is possible to type manually guid (negative number), but indeed it is not possible to choose from `creature` list right now. I have to add it
2. This is waiting on my todo list https://github.com/BAndysc/WoWDatabaseEditor/issues/10, however rather low priority as very little people are using latest wow
3. Do you mean dark theme? Yes, indeed it is hard to read, sadly the component for displaying SQL doesn't have good themes support so it is more work that just adjusting the colors, but will try to fix that as soon as possible!

Thanks again
